share/src/main/java/io/zensoft/share/service/facebook/FacebookUser.java

```diff
@@ -0,0 +1,45 @@
+package io.zensoft.share.service.facebook;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Service
+public class FacebookUser {
```

Comment:
I thought it was model class before i opened it

Reply:
It really looks like model. However, the purpose of this class is not to store data, but provide an access token of the proper page by given user access token.

Reply:
Then can you name it properly

Reply:
ok